template-for-repository/proofs/arpa-test.sh

```diff
@@ -0,0 +1,342 @@
+#!/bin/bash
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# This script performs  data collection for Arpa.
+# Furthermore, it serves as a validation framework for existing CBMC proofs.
+
+# TODO add project-specific list of proofs that fail, but should be overriden due to limitations of the validation approach
+# The list below must be seperated by spaces
+manual_proof_override=""
+
+
+function initialize {
+    # variable initializations
+    results=arpa-test-results.log
+    arpa_log=arpa-test-logs
+    t_pfs_w_unnesc=0
+    tot_prfs=$(ls -l | grep -c ^d)
```

Comment:
I prefer readable names like total_proofs.  

Is this supposed to be the total number of proofs in the proof repository?  This assumes all proofs are at the same level in the file system hierarchy.  This is incorrect in all the FreeRTOS work.